app/helpers/job_applications_helper.rb

```diff
@@ -1,11 +1,14 @@
 module JobApplicationsHelper
+  JOB_APPLICATION_DISPLAYED_STATUSES = %i[submitted unsuccessful shortlisted interviewing offered].freeze
```

Comment:
I think this constant suggests that the 'job offered' logic belongs possibly in a different partial - it seems to have altered the 'candidates' partial (and the controller) a bit too much for my liking